backend/accounts.go

```diff
@@ -205,6 +209,94 @@ func (backend *Backend) SupportedCoins(keystore keystore.Keystore) []coinpkg.Cod
 	return availableCoins
 }
 
+// AccountsByKeystore returns a map of the current accounts of the backend, grouped
+// by keystore.
+func (backend *Backend) AccountsByKeystore() (KeystoresAccountsListMap, error) {
+	defer backend.accountsAndKeystoreLock.RLock()()
+	accountsByKeystore := KeystoresAccountsListMap{}
+	for _, account := range backend.accounts {
+		persistedAccount := account.Config().Config
+		rootFingerprint, err := persistedAccount.SigningConfigurations.RootFingerprint()
+		if err != nil {
+			return nil, err
+		}
+		keystore, err := backend.Config().AccountsConfig().LookupKeystore(rootFingerprint)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Not sure what these lines are doing. Seems it is only used for `keystore.RootFingerprint`, but you already have `rootFingerprint` here.

Reply:
Right :see_no_evil: I missed it, thanks